Update Amazon EC2 Provider Permissions

cbudz opened the pull request:
Updated the EC2 provider permissions doc to reflect recommendations for admin group, as well as added new section on manual role creation for workaround.

cbudz commented:
Hi @dayleparker,

Please see the changes and file I've added here that addresses the workaround. Let me know what you think.

Thanks,
Chris

dayleparker commented on doc-Managing_Providers/topics/Amazon_EC2_Manual_Role_Creation.adoc:
```diff
@@ -0,0 +1,12 @@
+.. `AmazonEC2FullAccess`
+.. `AmazonS3FullAccess`
+.. `AmazonSQSFullAccess`
+. Input "smartstate" for the *Role name*. 
```
Should smartstate be entered with the quote marks? I think it might be more clear with this parameter marked up (with or without quotation marks)

dayleparker commented on doc-Managing_Providers/topics/Amazon_EC2_Providers_Permissions.adoc:
```diff
@@ -5,6 +5,11 @@
+ 
+[NOTE]
+====
+When adding an Amazon EC2 provider in {product-title_short} with the intention to use the SmartState Analysis feature, Red Hat recommends using the _Admin_ group.  For situations in which assigning the _Admin_ group is unacceptable, manually create an Amazon EC2 policy role using specific permissions. See xref:amazon-ec2-manual-role-creation[] for more information.
```
Would it be ok to delete 'Red Hat recommends" here to be more directive? Otherwise we will need to conditionalize this note for cfme...
One tiny nit - SmartState analysis uses a lowercase 'a' in the Glossary.

cbudz commented:
Hi @dayleparker 

I've added conditionalization to the doc, and corrected the capitalization for SmartState analysis. Markup has been applied in the Manual Creation doc, as well.

-Chris

dayleparker commented:
Thanks for clearing up those last pieces Chris, it all looks great :)
